### DISCUSSION TITLE
[Question][Hw3]

### DISCUSSION THREAD

**Discussion by ee12ha0220** [prosyslab-classroom/cs348-information-security#518] [Q&A]
In main.ml, where is "Llvm_irreader" defined?

**Comment by RiceBiscuits** [comment]
Llvm_irreader is one of LLVM modules. Therefore, it is defined at LLVM. You can check more details about Llvm_irreader at a link below. 
(https://llvm.moe/ocaml/Llvm_irreader.html)

**Comment by ee12ha0220** [comment]
Thank you! :)
